docs/constants/navigation.js

```diff
@@ -388,14 +389,12 @@ module.exports = {
 
 // --- MDX methods ---
 
-function makeSection(name, children = []) {
-  // TODO(cedric): refactor node types to match unist
-  return { name, children };
+function makeSection(name, children = [], props = {}) {
+  return make('section', { name, ...props }, children);
```

Comment:
These make methods are now not that necessary with the use of `unist-builder`. But, I kept it in to minimize the PR changes.